README.md

```diff
@@ -46,3 +46,172 @@ note: the UI currently uses code directly from github, pointing to the ts-client
+
+*Solution*:
+Make sure that instruction layout (defined using Anchor) used by client/cpi matches the program version.
+
+
+#### Program failed to complete: Access violation in stack frame 5 at address 0x200005ff8 of size 8 by instruction #6340
+No real solution yet! Please open an issue if you know how to fix this.
+
+*Workaround*:
+Experiment with `#[inline(never)]` on methods.
+
+
+#### instruction tries to borrow reference for an account which is already borrowed
+(*anchor-related issue*)
+
+*Solution*:
+Revisit the account references retrieved by explicit calls to `.load`/`.load_mut`. See doc on [`account_loader.rs`](https://github.com/coral-xyz/anchor/blob/fc9fd6d24b9be84abb2f40e47ed3faf7b11864ae/lang/src/accounts/account_loader.rs#L34).
+
+
+#### Running on-chain program crash silently on VM when using rust logging
+
+*Solution*:
+For Solana programs you must use `msg!` for logging and ***not*** `info!`/`debug!`/`error!`/`warn!`/`trace!` from the `log` crate.
+
+
+#### Exceeded max BPF to BPF call depth of 64 at instruction #4605 (stack size exceeded)
+
+*Solution*: Fix the Rust version - see [Section about Rust version](#wrong-rust-version)
+
+
+#### Program failed to complete: Invoked an instruction with data that is too large (12884928151 > 10240)
+
+*Solution*:
+Fix the Solana version (1.15.2 worked).
+
+
+#### Error: Function _ZN86_$LT$switchboard_v2..aggregator..AggregatorAccountData$u20$as$u20$core..fmt..Debug$GT$3fmt17h22734c1ad9ed3ea8E Stack offset of 4128 exceeded max offset of 4096 by 32 bytes, please minimize large stack variables
+This error results from very large Rust structures passed as parameters.
+
+No solution yet! Please open an issue if you know how to fix this.
+
+As of now the problem can be ignored as long as the method is not called.
+
+
+#### Syscall lib binding fails for invalid solana version combinations
+
+*Solution*:
+Make sure the Solana version of the program is compatible with the validator version.
+
+Check the Solana Feature Gate status and the [Solana Feature Gate Activation Schedule](https://github.com/solana-labs/solana/wiki/Feature-Gate-Activation-Schedule):
+```
+solana feature status
+
+# Output (truncated):
+Feature                                      | Status                  | Activation Slot | Description
+7rcw5UtqgDTBBv2EcynNfYckgdAaH1MAsCjKgXMkN7Ri | active since epoch 516  | 217388260       | enable curve25519 syscalls
+5x3825XS7M2A3Ekbn5VGGkvFoAg5qrRWkTrY4bARP1GL | inactive                | NA              | enable bpf upgradeable loader SetAuthorityChecked instruction #28424
+
+Tool Feature Set: 4033350765
+Software Version                    Feature Set   Stake     RPC
+1.17.1, 1.17.0                       2241946265   2.20%   1.75%
+1.16.17, 1.16.16, 1.16.15, 1.16.14   4033350765   9.12%   2.56%  <-- me
+1.16.13                              3949673676   0.41%   0.27%
+```
+
+
+
+
+#### Large Cpi Contexts (e.g. Serum) will eventually overflow the stack
+No real solution yet!
```

Comment:
Can you explain what this means further?

Reply:
done